kin-sdk/kin-sdk-lib/src/androidTest/java/kin/sdk/AppIdTest.java

```diff
@@ -0,0 +1,74 @@
+package kin.sdk;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+import static kin.sdk.IntegConsts.TEST_NETWORK_ID;
+import static kin.sdk.IntegConsts.TEST_NETWORK_URL;
+
+import android.support.test.InstrumentationRegistry;
+import android.util.Log;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class AppIdTest {
+
+    private static final String APP_ID = "1a2c";
+    private static final String STORE_KEY_TEST = "test";
+
+    private KinClient createNewKinClient(String appId) {
```

Comment:
I think it can be unit test, don't see a reason to have it as android test.
(we have already test harness for unit testing kin client in `KinClientTest`)

Reply:
ahh, you can get away with Robolectric in this case, (we're using it already here and there),
but as we have a special ctor for tests that are not calling `validateMemo` it doesn't really matter.